### PR TITLE
feat: chart reload button on error

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

The requests to the CryptoCompare API fail occasionally, causing console errors and without providing feedback to the user. This PR adds error handling and shows a notice if the chart data failed to load correctly.

image

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

**Review by faustbrian — APPROVED** [review]

**Comment by ItsANameToo** [comment]
@faustbrian comments regarding these changes, as they are just to understand where calculations might go wrong:

1. Delegate list order seems to be off by 21, so shifting the ranks by 21 before returning it as `activeDelegates` fixes that aspects (question remains how this can be wrong)
2. By changing the order above, the time to forge is incorrectly shown for the newly ordered delegates due to slots still matching the old order (why??), tried to remedy that by shifting the index too but that doesn't really solve it

so tl;dr looks like slots are wrongly calculated? with the same weird offset of 21 as the original `activeDelegates` list

**Comment by ItsANameToo** [comment]
@faustbrian some smaller changes for now that had to be fixed anyway

**Comment by faustbrian** [comment]
@ItsANameToo conflicts

**Comment by ItsANameToo** [comment]
@faustbrian where?
